cpp/src/sender.cpp

```diff
@@ -66,8 +66,15 @@ uint64_t tag_counter = 0;
 
 tracker sender::send(const message &message) {
     uint64_t id = ++tag_counter;
-    pn_delivery_t *dlv =
-        pn_delivery(pn_object(), pn_dtag(reinterpret_cast<const char*>(&id), sizeof(id)));
+    const binary tag(
+        std::string(reinterpret_cast<const char *>(&id), sizeof(id)));
```

Comment:
I don't much like bringing in `std::string`. What about this?

```
    const uint8_t *begin = reinterpret_cast<const uint8_t *>(&id);
    const binary tag(begin, begin + sizeof(id));
```

Reply:
Oh okay!! Thank you. Done.